examples/django_app/example_app/static/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Since the Bootstrap Glyphicons aren't actually used in the example app, I'd rather not add them. I realize that a message about them not being found is currently displayed in the networking output. It might be better to just remove the line in the stylesheets that loads the Glyphicons.